firstvoices/firstvoices/settings.py

```diff
@@ -241,6 +241,8 @@
 CELERY_RESULT_BACKEND = os.getenv("CELERY_RESULT_BACKEND", "redis://localhost/0")
 CELERY_BEAT_SCHEDULER = "django_celery_beat.schedulers:DatabaseScheduler"
 CELERY_ALWAYS_EAGER = True
+# Celery tasks are not picked up by autodiscover_tasks() if they are not globally imported. This adds missing tasks.
+CELERY_IMPORTS = ("backend.tasks.update_metadata_tasks",)
```

Comment:
Thanks for the info. I will add search related tasks as well to the celery imports list after your PR is merged.

Reply:
I was having an issue where the task could not be found when importing in a management command. 

I don't believe this variable makes the worker auto-restart after changes.

It looks like there is a now deprecated auto-restart flag for celery but there may be a way to add a management command that starts the celery worker with auto-reload working. See [this link](https://stackoverflow.com/a/49166246) for info.